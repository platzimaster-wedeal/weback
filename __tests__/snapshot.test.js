const getCharacter = require('../api/snapshot.example')
const rick = require('../rick.json')

describe('ItsTimeToSnapshots', () => {
  test('SnapshotOfRick', () => {
    expect(getCharacter(rick)).toMatchSnapshot()
  })
  test('EverWillFailSnapshot', () => {
    const user = {
      createAt: new Date(),
      id: Math.floor(Math.random() * 10)
    }
    expect(user).toMatchSnapshot()
  })
  test('WeHaveException', () => {
    const user = {
      id: Math.floor(Math.random() * 10),
      name: 'Rub√©n Esparza'
    }
    expect(user).toMatchSnapshot({
      id: expect.any(Number)
    })
  })
})
