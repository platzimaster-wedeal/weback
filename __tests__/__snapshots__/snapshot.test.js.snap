// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItsTimeToSnapshots EverWillFailSnapshot 1`] = `
Object {
  "createAt": 2020-09-09T19:16:31.836Z,
  "id": 1,
}
`;

exports[`ItsTimeToSnapshots SnapshotOfRick 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`ItsTimeToSnapshots WeHaveException 1`] = `
Object {
  "id": Any<Number>,
  "name": "Rub√©n Esparza",
}
`;
